raft/log.go

```diff
@@ -455,6 +482,14 @@ func (l *Log) Initialize() error {
 		return err
 	}
 
+	// Wait until leader state change.
```

Comment:
This section actually isn't needed any more since `startStateLoop()` handles the wait. It was meant to wait until the state changed to `Leader`.